sqls/test3.sql

```diff
@@ -0,0 +1,7 @@
+WITH cte AS (
+    SELECT * FROM foo
+)
+
+SELECT a, b FROM cte
+UNION
+SELECT a, b FROM t
```

Comment:
🚫 **[sqlfluff]** <[LT09](https://docs.sqlfluff.com/en/stable/rules.html#rule-LT09)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[layout.select_targets] - Select targets should be on a new line unless there is only one select target.
```suggestion
SELECTa, b FROM t
```
```suggestion
SELECT 
a, b FROM t
```
```suggestion
SELECT a,b FROM t
```
```suggestion
SELECT a, 
b FROM t
```
```suggestion
SELECT a, bFROM t
```
```suggestion
SELECT a, b 
FROM t
```

<!-- __reviewdog__:ChA1YjA5MTQyMmI2MjU5MzIzEghzcWxmbHVmZg== -->